packages/components/tooltip/src/tooltip.js

```diff
@@ -53,10 +71,14 @@ const Tooltip = (props) => {
     };
   }, []);
 
-  const { reference, popper } = usePopper({
+  const [referenceElement, setReferenceElement] = React.useState(null);
+  const [popperElement, setPopperElement] = React.useState(null);
```

Comment:
Shouldn't we use `useRef`? 🧐

Reply:
It does seem odd, but that's how it's done in the official documentation:
https://popper.js.org/react-popper/v2/#example